test/sequential/test-regress-fs-stat_sync.js

```diff
@@ -22,6 +22,14 @@
 'use strict';
 require('../common');
 const { fixturesDir } = require('../common/fixtures');
+
+// Check that the calls to Integer::New() and Date::New() succeed and bail out
```

Comment:
This file can be named `test-fs-statSync-overflow.js` since it is testing that specific behavior of `fs.statSync`.

Also, can you rename the fixtures used below as well? Thanks